usecases/schema/manager_test.go

```diff
@@ -532,6 +533,33 @@ func Test_ParseVectorConfigOnDiskLoad(t *testing.T) {
 	}, classes[0].VectorIndexConfig)
 }
 
+func Test_DestinationPath(t *testing.T) {
+	logger, _ := test.NewNullLogger()
+
+	repo := newFakeRepo()
+	repo.schema = &State{
+		ObjectSchema: &models.Schema{
+			Classes: []*models.Class{{
+				Class:             "Foo",
+				VectorIndexConfig: "parse me, i should be in some sort of an object",
```

Comment:
I think that `VectorIndexConfig` is of `map[string]interface{}` type not a `string`